README.md

```diff
@@ -84,8 +109,12 @@ software by running *make install* as priviledged user.
     ...
     $# make install
 
+    or : 
+    $> make && make install
+
 You're done !
 
+
```

Comment:
Verify /etc/centreon-engine directory.

Usually, its content is filled by the Centreon web interface, owned by the apache user.

A command like the following may be necessary:

chown -R apache:apache /etc/centreon-engine/*